cmd/podman/system/reset.go

```diff
@@ -43,6 +51,15 @@ func init() {
 	})
 	flags := systemResetCommand.Flags()
 	flags.BoolVarP(&forceFlag, "force", "f", false, "Do not prompt for confirmation")
+
+	flags.StringVar(&givenUser, "user", "", "User to setup the DB under")
```

Comment:
What is the use of user?  I don't think this should be supported.

Reply:
User is kind of a quick filler option where if you can just specify something like `--user=charliedoern` and storage.conf will be populated with working paths if you broke it by putting some weird options.

Reply:
Remove it, if I want to do that I can just do

sudo charliedoern podman system reset

Reply:
Ok, should we then remove storage.conf if `-f` is specified? If we don't remove the old storage.conf set up by the `--run-root` or `--graph-root` options then those will be used. Or I guess we can always write the new runtime config to storage.conf if it exists?

Reply:
--force flag has nothing to do with this. storage.conf should only be rewritten if user specifies the --run-root and --graph-root options.